pacman/makepkg_clang32.conf

```diff
@@ -59,9 +59,9 @@ DXSDK_DIR=${MINGW_PREFIX}/${MINGW_CHOST}
 CPPFLAGS="-D__USE_MINGW_ANSI_STDIO=1"
 CFLAGS="-march=i686 -mtune=generic -O2 -pipe"
 CXXFLAGS="-march=i686 -mtune=generic -O2 -pipe"
-LDFLAGS="-pipe"
-# Uncomment to enable hardening (ASLR, DEP)
-#LDFLAGS="-pipe -Wl,--dynamicbase,--nxcompat"
+LDFLAGS="-pipe -Wl,--dynamicbase,--nxcompat,--no-seh"
```

Comment:
LLD silently ignores `--nxcompat` because it's always enabled.
Do we still want to include it?

Reply:
Maybe we can keep that just for similarity with makepkg_mingw* one 🤷

Reply:
ok, I'll remove it. I made sure lld doesn't fail with them, but I didn't know if they were useful.

Reply:
@Biswa96 we want to get rid of them in makepkg_mingw as well eventually, once they are default in binutils, so this is fine imo